ext/strings.go

```diff
@@ -34,45 +34,52 @@ import (
 //
 // CharAt
 //
-// Returns the character at the given position. If the position is less than 0 or greater than the
-// length of the string, the function will produce an error. CharAt is an instance method with the
-// following signature:
+// Returns the character at the given position. If the position is negative, or greater than
```

Comment:
It's probably worth having a package-wide comment that indexes are zero-based throughout, rather than replicate that statement everywhere.

Reply:
Done, not as a package comment, but at the top of the comment block for the `Strings` function.